### ISSUE TITLE
VisualizerTask breaks test execution

### ISSUE THREAD

**Issue by siad007** [phingofficial/phing#1245]
For some reasons the VisualizerTask breaks the test execution with:
```
BUILD FAILED

Test ERROR (testFormat in class VisualizerTaskTest /home/travis/build/phingofficial/phing/classes/phing/Task.php on line 251): Malformed response:
```
or
```
Test ERROR (testDefaultValues in class VisualizerTaskTest /home/travis/build/phingofficial/phing/classes/phing/tasks/ext/visualizer/VisualizerTask.php on line 427): Request unsuccessful. Response from server: 504 Gateway Time-out
```
@jawira would you mind to have a look here?

**Comment by jawira** [comment]
Not sure where is the problem. When I execute the tests I don't have any issue (of course).
Could you send me the link to the failing Travis job?

Anyway, I assume the error comes from external PlantUML servers which are not reliable. These server are used to generate the call graph in _png_, _svg_ and _eps_ formats. However PlantUML servers are not required to generate diagrams in _puml_ format.

So, what about removing all the tests that requires an external server?

**Comment by siad007** [comment]
Sounds good!
BTW the last failing for a couple of minutes ago:
https://travis-ci.org/phingofficial/phing/jobs/643045286?utm_medium=notification&utm_source=github_status

**Comment by siad007** [comment]
Fixed in #1247
